core/src/test/java/org/elasticsearch/index/translog/TranslogTests.java

```diff
@@ -1008,42 +1032,6 @@ public void testTranslogWriter() throws IOException {
         IOUtils.close(writer);
     }
 
-    public void testFailWriterWhileClosing() throws IOException {
```

Comment:
This test was testing a specific scenario: constructing the new translog reader could fail because we did a file operation (reading the channel position) in that construction. That file operation was removed, so that failure scenario can not occur anymore. Are you saying that you want the test to remain, just without testing for the failure scenario?